pkg/provider/github/github.go

```diff
@@ -173,19 +173,46 @@ func makeClient(ctx context.Context, apiURL, token string) (*github.Client, stri
 	return client, providerName, github.String(apiURL)
 }
 
+func parseTS(headerTS string) (time.Time, error) {
+	ts := time.Time{}
+	// Github go-github library as defined there
+	if t, err := time.Parse("2006-01-02 03:04:05 MST", headerTS); err == nil {
+		ts = t
+	}
+
+	// Normal UTC: 2023-01-31 23:00:00 UTC
+	if t, err := time.Parse("2006-01-02 15:04:05 MST", headerTS); err == nil {
+		ts = t
+	}
```

Comment:
The 2nd parse includes this as the difference is just that it supports hours 0-23 instead of just 0-11 as `go-github` does.

```suggestion
	// Normal UTC: 2023-01-31 23:00:00 UTC
	if t, err := time.Parse("2006-01-02 15:04:05 MST", headerTS); err == nil {
		ts = t
	}
```